pkg/tracing/container.go

```diff
@@ -0,0 +1,30 @@
+package tracing
+
+import (
+	"fmt"
+	"os"
+
+	runtimeclient "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/runtime-client"
+	containerutilsTypes "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/types"
+	igtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+)
+
+func DetectContainerRuntime(hostMount string) (*containerutilsTypes.RuntimeConfig, error) {
+	runtimes := map[igtypes.RuntimeName]string{
```

Comment:
inspektor gadget support only in default location?
How can we know what runtime is in use by K8s? I have docker + podman...

Reply:
See my latest push, I have changed this so it can be overwritten with environment variables